pkg/kubeapiserver/authorizer/modes/modes_test.go

```diff
@@ -20,26 +20,29 @@ import "testing"
 
 func TestIsValidAuthorizationMode(t *testing.T) {
 	var tests = []struct {
+		name      string
 		authzMode string
 		expected  bool
 	}{
-		{"", false},
-		{"rBAC", false},        // not supported
-		{"falsy value", false}, // not supported
-		{"RBAC", true},         // supported
-		{"ABAC", true},         // supported
-		{"Webhook", true},      // supported
-		{"AlwaysAllow", true},  // supported
-		{"AlwaysDeny", true},   // supported
+		{"test-no value", "", false},
```

Comment:
why do we need this redundancy of data? Can't we use the authzMode and make a string out of it as test name?

Reply:
I think if you just want to get a unique string, in fact, the prefix with test is actually not necessary, here is mainly to show that this is a test case.